app/models/assignment.rb

```diff
@@ -46,4 +50,12 @@ def sibling_assignments
       .where.not(id: id)
       .where.not(user: user_id)
   end
+
+  private
+
+  def set_defaults_and_normalize
+    self.article_title = article_title.tr(' ', '_') unless article_title.nil?
+    # FIXME: transitional only
+    self.wiki_id ||= Wiki.default_wiki.id
+  end
```

Comment:
A more robust approach would be to use `Utils.format_article_title`, which also takes care of capitalizing the first letter.